frontend/src/utils/CampUtils.ts

```diff
@@ -159,3 +205,88 @@ export const getSessionBorderColor = (sessionIndex: number): string =>
 
 export const getSessionFillColor = (sessionIndex: number): string =>
   FILL_COLORS[sessionIndex % 8];
+
+// isEDLPValid checks if the EDLP config in the request object is valid
+const isEDLPValid = (camp: CreateUpdateCampRequest): boolean => {
+  // Camp either doesn't have EDLP or has both
+  if (
+    (camp.earlyDropoff && !camp.latePickup) ||
+    (camp.latePickup && !camp.earlyDropoff)
+  ) {
+    return false;
+  }
+  // Camp must have both late pickup and early dropoff fee if it has EDLP
+  if (
+    camp.earlyDropoff &&
+    camp.latePickup &&
+    (!camp.dropoffFee || !camp.pickupFee)
+  ) {
+    return false;
+  }
+  return true;
+};
+
+// isMinCampDetailsFilled checks if the camp object has the minimum details required to create/update a camp
+export const isMinCampDetailsFilled = (
+  camp: CreateUpdateCampRequest,
+): boolean => {
+  if (
+    camp.name &&
+    camp.description &&
+    camp.fee &&
+    camp.startTime &&
+    camp.endTime &&
+    camp.ageLower &&
+    camp.ageUpper &&
+    isEDLPValid(camp) &&
+    camp.location.streetAddress1 &&
+    camp.location.city &&
+    camp.location.province &&
+    camp.location.province &&
+    camp.location.postalCode &&
+    camp.campSessions.length > 0 &&
+    camp.campSessions.every((cs) => cs.capacity > 0)
+  ) {
+    return true;
+  }
+  return false;
```

Comment:
Can we just return the condition directly, without the explicit `if` and `return` statements?

Reply:
having just a return statement requires explicitly checking for empty string and 0 for nums etc. since the && operator doesn't work with different types. Thought at that point that the if statement was cleaner